pom.xml

```diff
@@ -18,6 +18,14 @@
         </maven.checkstyle.plugin.configLocation>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>com.mysql</groupId>
+            <artifactId>mysql-connector-j</artifactId>
```

Comment:
```suggestion
            <artifactId>mysql-connector-java</artifactId>
```
Did it work?

Reply:
Yes)

Reply:
![Screenshot_3](https://github.com/mate-academy/jv-jdbc-intro/assets/151533079/2f2a177a-38da-420e-a8a4-eedb762c7c13)

Reply:
If use full name doesn t work)